packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -396,6 +418,14 @@ class NavigatorState extends State<Navigator> {
     _cancelActivePointers();
   }
 
+  /// Replaces a route that is not currently visible with a new route.
+  ///
+  /// The new route and the route before the new route (if any) are notified
```

Comment:
The meaning of 'the route before' isn't crystal clear. Since this is a stack model, then 'route below'